source/ios/tools/IOSFeed/IOSFeed.cs

```diff
@@ -0,0 +1,146 @@
+                CloudBlobClient cloudBlobClient = storageAccount.CreateCloudBlobClient();
+
+                CloudBlobContainer cloudBlobContainer =
+                    cloudBlobClient.GetContainerReference(Constants.ContainerId);
+                await cloudBlobContainer.CreateIfNotExistsAsync();
+
+                // Set the permissions so the blobs are public.
+                BlobContainerPermissions permissions = new BlobContainerPermissions
+                {
+                    PublicAccess = BlobContainerPublicAccessType.Blob
+                };
+
+                await cloudBlobContainer.SetPermissionsAsync(permissions);
+
+                var sourceFile = Path.Combine(Constants.FrameworkPath, Constants.FrameworkName);
+                var blobGuid = Guid.NewGuid().ToString();
+                var cloudFileName = blobGuid + Constants.FrameworkName;
+
+                CloudBlockBlob cloudBlockBlob = cloudBlobContainer.GetBlockBlobReference(cloudFileName);
+
+                await cloudBlockBlob.UploadFromFileAsync(sourceFile);
+
+                BlobContinuationToken blobContinuationToken = null;
+                do
+                {
+                    var results = await cloudBlobContainer.ListBlobsSegmentedAsync(null, blobContinuationToken);
+                    // Get the value of the continuation token returned by the listing call.
+                    blobContinuationToken = results.ContinuationToken;
+                    foreach (IListBlobItem item in results.Results)
+                    {
+                        var uriString = item.Uri.ToString();
+                        if (uriString.Contains(blobGuid))
+                        {
+                            UpdatePodSpec(uriString);
+                        }
+                    }
+                } while (blobContinuationToken != null);
+            }
+            else
+            {
+                throw new InvalidOperationException("invalid connection string");
+            }
+        }
+
+        private static void UpdatePodSpec(string uri)
+        {
+            var sourceFile = Path.Combine(Constants.PodspecPath, Constants.PodspecName);
+            var targetFile = Path.Combine(Constants.TargetPodspecPath, Constants.PodspecName);
+
+            FileInfo fi = new FileInfo(sourceFile);
+            if (!fi.Exists)
+            {
+                throw new FileNotFoundException("missing podspec");
+            }
+
+            // Open the stream and read it back.
+            using (StreamReader sr = File.OpenText(sourceFile))
+            {
+                string s = "";
+                string output = "";
+                while ((s = sr.ReadLine()) != null)
+                {
+                    if (s.Length != 0)
+                    {
+                        var splits = s.Split('=');
+                        if (splits.Length > 0)
+                        {
+                            if (splits[0].Contains("spec.source"))
```

Comment:
Nit: Note that string.Contains() is a case-sensitive comparison by default - do you want it to be case sensitive or just make it case-insensitive (OrdinalIgnoreCase)

Reply:
thanks

---
In reply to: [314079012](https://github.com/microsoft/AdaptiveCards/pull/3291#discussion_r314079012) [](ancestors = 314079012)